App/Nevermore.lua

```diff
@@ -182,20 +179,20 @@ local lib = {}
 -- @function LoadLibrary
 -- @tparam Variant LibraryName Can either be a ModuleScript or string
```

Comment:
Out of curiosity, what is the implication of loading the same library twice, once with its name, and once with the ModuleScript reference?

Reply:
It should function exactly the same. Roblox handles the caching.